app/templates/admin/content/events.hbs

```diff
@@ -15,31 +15,33 @@
           </div>
         </div>
         <div class="ui divider"></div>
-        <table class="ui celled table">
-          <tbody>
-            {{#each model.eventTypes as |eventType|}}
-              <tr>
-                <td>
-                  <div class="ui grid">
-                    <div class="middle aligned content eight wide column left aligned">
-                      {{eventType.name}}
-                    </div>
-                    <div class="eight wide column right aligned">
-                      <div class="ui horizontal compact basic buttons">
-                        {{#ui-popup content=(t 'Edit') class='ui icon button' click=(action 'openModalFor' eventType)}}
-                          <i class="edit icon"></i>
-                        {{/ui-popup}}
-                        {{#ui-popup content=(t 'Delete') class='ui icon button' click=(action (confirm (t 'Are you sure you would like to delete this event type?') (action 'deleteEventProperty' eventType)))}}
-                          <i class="trash outline icon"></i>
-                        {{/ui-popup}}
+        <div  class = "y-scrollable" style = "max-height: 768px;overflow-x: hidden;">
+          <table class="ui celled table">
+            <tbody>
+              {{#each model.eventTypes as |eventType|}}
```

Comment:
Avoid custom css, and I guess using the `y-scrollable` class in in `<table>` should work fine something like this `<table class = "ui celled table y-scrollable">` . Have you tried it?

Reply:
> Avoid custom css, and I guess using the `y-scrollable` class in in `<table>` should work fine something like this `<table class = "ui celled table y-scrollable">` . Have you tried it?

I have tried many ways , working on it around 3 days to find a menthid that would not use custom css but max-height is the problem . without using max-height how i am supposed to fix height above which table should be scrollable.